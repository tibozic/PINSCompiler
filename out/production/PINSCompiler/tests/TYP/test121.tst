fun f(x: integer, y: logical, z: logical, k: string): integer = ({for x=x,x,y:x},x)
