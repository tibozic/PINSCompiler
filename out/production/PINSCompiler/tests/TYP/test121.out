[1:77-1:78]: ERROR: For loop step type has to be an integer not `log`

